railties/lib/rails/generators/app_base.rb

```diff
@@ -404,6 +412,49 @@ def run_bundle
         bundle_command("install") if bundle_install?
       end
 
+      def npm_path
+        commands = ["npm"]
```

Comment:
I think we can add also to "yarn" command here (https://code.facebook.com/posts/1840075619545360)